packages/amplication-server/src/core/serviceSettings/serviceSettings.service.ts

```diff
@@ -61,6 +60,8 @@ export class ServiceSettingsService {
       );
     }
 
+    serviceSettings.serverSettings.generateServer = true;
+
```

Comment:
you moved this line without moving its comment (line 54)

Reply:
Okay, I will make a new PR.

Reply:
@abrl91 The PR :- https://github.com/amplication/amplication/pull/7807